raiden/transfer/log.py

```diff
@@ -85,6 +85,30 @@ def __init__(self, database_path):
         )
         self.conn.commit()
 
+        self.sanity_check()
+
+    def sanity_check(self):
+        """ Ensures that NUL character can be safely inserted and recovered
```

Comment:
I have never seen it abbreviated like that. If you really don't want to add that it's fine.